design/src/high-level/game-thesis.md

```diff
@@ -0,0 +1,176 @@
+- **Technology**
+  - guided evolution: research analogue used to modify and enhance existing species
+  - assimilation: research analogue used to add new species to the colony
+- **Production enhancements**
+  - pheromones: spent manually by the player to temporarily and locally enhance
+  - upgrades: done via guided evolution, affecting all organisms of that strain
+  - enhancements: pheromones can be produced and applied automatically via workers
+
+### Supplementary Mechanics
+
+These features supplement the core gameplay loop by providing additional things to do or consider, but are not needed.
+
+- **Exploration**
+  - important but not essential
+  - fairly standard implementation
+  - needs more interesting world generation
+  - water distribution and topography are key
+- **Combat**
+  - natural but not essential
+  - must be careful to avoid snowballing effects
+    - design with negative feedback loops like hunger satiation
+- **Pollution**
+  - thematically essential
+  - needs much richer (and more plausible) model of pollution
+  - effects should be varied and depend on intensity of pollution
+  - polluting should be easier than dealing with waste products properly
+
+## Design Strengths
+
+- biology serves as a great source of easy-to-explain inspiration for unique game mechanics
+- compelling and unique thematics
+  - opportunities for political/moral storytelling on importance of sustainability and dangers of pollution
+- disruptions offer a unique opportunity for players to explore more robust factory designs
+- disruptions can create much more intresting emotional pacing in a genre that struggles with flatness
+- tiny scale offers interesting mechanical and experiential possibilities that will feel new and interesting
+- emphasis on sustainability pushes designers and players towards more interesting resource refinement pathways
+- assimilating new species offers a natural and high impact path to adding more options for players
+  - this can be done as horizontal progression, allowing new players to jump into whatever interests them most
+- pheromones offer the player more ability to respond to the world (at a cost), and a more engaging gameplay experience
+- lack of player avatar reduces frustration of inventory management and walking around
+- modded Factorio has really nailed much of the UX and QoL features that we needed, and offers a clear base to learn from
+
+## Design Challenges
+
+- trap of realism
+  - the availability of ecology and biology and ecology risks designers targeting realism over player experience
+  - the world must be **plausible**, not realistic
+  - for example, realistic evolution systems are likely to be fiddly and frustrating and distracting, rather than rich and interesting
+  - some audience segments may be upset by "unrealistic" mechanics
+    - mods are a helpful outlet for them
+- scope creep due to abundance of fascinating ideas
```

Comment:
Yes, agreed! I think an important next step in game design will be codifying an agreed upon baseline system of organisms and features. 

This would be especially important for programming the simulation. Currently, I am making things up as I go, but the concern with that is that I might have to heavily rework things as I figure out: "oh, that's not quite what we wanted to do"

(Recent example: I have been treating food as a discrete resource, as is commonly done in various games.)